components/Alert/Alert.tsx

```diff
@@ -5,7 +5,7 @@ import * as React from 'react';
 import { cva, type VariantProps } from 'class-variance-authority';
 
 const alertVariants = cva(
-  'relative w-full rounded-lg border p-4 [&>svg~*]:pl-7 [&>svg+div]:translate-y-[-3px] [&>svg]:absolute [&>svg]:left-4 [&>svg]:top-4 [&>svg]:text-foreground',
+  'relative w-full max-w-[25rem] rounded-lg border p-4 [&>svg~*]:ml-7 [&>svg+div]:translate-y-[-3px] [&>svg]:absolute [&>svg]:left-4 [&>svg]:top-4 [&>svg]:text-foreground',
```

Comment:
Instead of using an arbitrary tailwind class, `max-w-[25rem]`, can you just use `max-w-96` which is equivalent to `24rem`?

Or, like I suggested, `max-w-fit` which is the same as `max-width: fit-content;`. This means all toasts won't be the same size, but will be larger/smaller based on the content inside.

You could also add a `min-w` if you wanted to ensure it would be _no less_ than a certain width.
max-width: fit-content;
--


<br class="Apple-interchange-newline">
max-width: fit-content;
--


<br class="Apple-interchange-newline">

Reply:
I was thinking that a consistent width might look better if there are multiple toasts stacked so that's why I set a `max-width` but I see the argument for either approach.

Reply:
@kepsteen I agree with the consistent width.